app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def calculate_team_total_rating(team: int) -> int:
```

Comment:
The parameter `team` should be a list of `Player` objects, not an integer. Update the type annotation to `List[Player]`.